playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -80,6 +80,17 @@ parameter_defaults:
   CephPoolDefaultPgNum: 8
   CephPoolDefaultSize: 1
 {% endif %}
+{% if ssl_enabled %}
+  AddVipsToEtcHosts: true
+  HorizonSecureCookies: True
+  SSLCertificate: |
+    {{ ssl_cert | indent }}
+  SSLKey: |
+    {{ ssl_key | indent }}
+  SSLRootCertificate: |
+    {{ ssl_ca_cert | indent }}
```

Comment:
Possibly these things live as a secret in `ci-configs` and we pass them as parameters to ansible.
In other words, we may need to allow passing an existing cert when deploying the OpenStack platform for our CI needs since it could be difficult to dynamically update the cert referenced by the `clouds.yaml` file in CI. To investigate.

Reply:
ack, I'll add support for this.